docs/src/development/environment-setup.md

```diff
@@ -13,39 +14,35 @@ If you already have Rust installed make sure to upgrade it by running
 $ rustup update
 ```
 
-To verify that project was cloned and set up correctly, you can run
+### Scarb
+You can read more about installing scarb [here](https://docs.swmansion.com/scarb/download.html)
 
-```shell
-$ cargo check
-```
+Please make sure you're using scarb installed via asdf - otherwise some tests may fail.
+> To verify, run:
+> 
+> ```shell
+> $ which scarb
+> $HOME/.asdf/shims/scarb
+> ```
+> 
+> If you previously installed scarb using official installer, you may need to remove this installation or modify your PATH to make sure asdf installed one is always used.
 
-## Running Tests
+### Starknet Devnet
+To install it run `./scripts/install_devnet.sh`
+
+### Universal sierra compiler
+Install the latest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler) version.
 
-> 📝 **Note**
-> 
-> Make sure you run `./scripts/install_devnet.sh`
-> and then set [Scarb](https://docs.swmansion.com/scarb/) version 
-> [compatible](https://github.com/foundry-rs/starknet-foundry/releases) with both `snforge` and `sncast`
-> and use the newest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler)
-> after setting up the development environment, otherwise the tests will fail.
 
+
+## Running Tests
+Make sure to install are prerequisites beforehand!
```

Comment:
Not useful imo, that's why they are called `prerequisites`

Reply:
Done